scripts/utils/starknet.py

```diff
@@ -246,12 +283,63 @@ def _convert_offset_to_hex(obj):
     )
 
 
+async def deploy_starknet_account(private_key=None, amount=1) -> Account:
+    compile_contract(
+        {"contract_name": "OpenzeppelinAccount", "is_account_contract": True}
+    )
+    class_hash = await declare("OpenzeppelinAccount")
+    salt = random.randint(0, 2**251)
+    global _private_key
+    global _account_address
+    private_key = private_key or _private_key
+    if private_key is None:
+        raise ValueError(
+            "private_key was not given in arg nor in env variable, see README.md#Deploy"
+        )
+    key_pair = KeyPair.from_private_key(int(private_key, 16))
+    constructor_calldata = [key_pair.public_key]
+    address = compute_address(
+        salt=salt,
+        class_hash=class_hash,
+        constructor_calldata=constructor_calldata,
+        deployer_address=0,
+    )
+    logger.info(f"ℹ️  Funding account {hex(address)} with {amount} ETH")
+    await fund_address(address, amount=amount)
+    logger.info(f"ℹ️  Deploying account")
+    res = await Account.deploy_account(
+        address=address,
+        class_hash=class_hash,
+        salt=salt,
+        key_pair=key_pair,
+        client=RPC_CLIENT,
+        chain=CHAIN_ID,
+        constructor_calldata=constructor_calldata,
+        max_fee=int(1e17),
```

Comment:
should we once and for all create a MAX_FEE const? and use it everywhere?

Reply:
possibly. tbh I don't remember why we don't use the fee estimation step and set max fee instead, I can try

Reply:
but also I had a tx failing with max_fee = 0.1 an not with 0.5, while the actual fee was much lower so the fee estimation is not really accurate. We'll need to clarify this when we launch mainnet (wen mainnet) but up until this point, these random hard coded values look ok to me we just to make sure that it passes

Reply:
alright